packages/js/tsconfig.json

```diff
@@ -13,10 +13,14 @@
 		"typeRoots": []
 	},
 	"exclude": [
+		"bin",
```

Comment:
Note for myself: the updated configs cover this already, doublecheck and revert if so.